Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -0,0 +1,241 @@
+        if let extentGeometry = plane.node.geometry as? SCNPlane {
+            extentGeometry.width = CGFloat(planeAnchor.extent.x)
+            extentGeometry.height = CGFloat(planeAnchor.extent.z)
+            plane.node.simdPosition = planeAnchor.center
+        }
+    }
+}
+
+/// Helper class to visualize a plane found by ARKit
+class Plane: SCNNode {
+    let node: SCNNode
+    
+    init(anchor: ARPlaneAnchor) {
+        // Create a node to visualize the plane's bounding rectangle.
+        let extent = SCNPlane(width: CGFloat(anchor.extent.x), height: CGFloat(anchor.extent.z))
+        node = SCNNode(geometry: extent)
+        node.simdPosition = anchor.center
+        
+        // `SCNPlane` is vertically oriented in its local coordinate space, so
+        // rotate it to match the orientation of `ARPlaneAnchor`.
+        node.eulerAngles.x = -.pi / 2
+
+        super.init()
+
+        node.opacity = 0.25
+        guard let material = node.geometry?.firstMaterial
+            else { fatalError("SCNPlane always has one material") }
+        
+        material.diffuse.contents = UIColor.white
+
+        // Add the plane node as child node so they appear in the scene.
+        addChildNode(node)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private extension SceneViewProxy {
+    /// Sets the initial transformation used to offset the originCamera.  The initial transformation is based on an AR point determined via existing plane hit detection from `screenPoint`.  If an AR point cannot be determined, this method will return `false`.
+    ///
+    /// - Parameter screenPoint: The screen point to determine the `initialTransformation` from.
+    /// - Returns: The `initialTransformation`.
+    /// - Since: 200.3
+    func setInitialTransformation(
+        for arViewProxy: ARSwiftUIViewProxy,
+        using screenPoint: CGPoint,
+        cameraController: TransformationMatrixCameraController
+    ) -> TransformationMatrix? {
+        // Use the `internalHitTest` method to get the matrix of `screenPoint`.
+        guard let matrix = internalHitTest(using: screenPoint, for: arViewProxy) else { return nil }
+
+        // Set the `initialTransformation` as the TransformationMatrix.identity - hit test matrix.
+        let initialTransformation = TransformationMatrix.identity.subtracting(matrix)
+
+        return initialTransformation
+    }
+    
+    /// Internal method to perform a hit test operation to get the transformation matrix representing the corresponding real-world point for `screenPoint`.
+    ///
+    /// - Parameter screenPoint: The screen point to determine the real world transformation matrix from.
+    /// - Returns: An `TransformationMatrix` representing the real-world point corresponding to `screenPoint`.
+    func internalHitTest(using screenPoint: CGPoint, for arViewProxy: ARSwiftUIViewProxy) -> TransformationMatrix? {
```

Comment:
I wouldn't include the prefix "internal" unless it's needed. I don't think it is in this case. 

Also this doesn't seem to need to extend SceneViewProxy. It might make more sense extending ARSwiftUIProxy.